react/ProductSummaryImage.tsx

```diff
@@ -440,6 +446,15 @@ function ProductImage({
               alt={name}
               className={imageClassname}
               onError={onError}
+              fetchpriority={
```

Comment:
Should the case of no `position` be handled here?

Reply:
Yes, I'll adapt the code for handling this case.